onnxruntime/contrib_ops/cuda/layer_norm.cc

```diff
@@ -24,7 +24,7 @@ namespace cuda {
       LayerNorm<T, U>);
 
 REGISTER_KERNEL_TYPED(float, float)
-REGISTER_KERNEL_TYPED(double, float)
+REGISTER_KERNEL_TYPED(double, double)
```

Comment:
>D(double, double) [](start = 20, length = 17)

Would it possible to combine these 3 in a single declaration? You can use BuildKernelDefConstraints() to combine so less typing.

Reply:
The kernel definition will be non-typed. But we need to handle type in the kernel implementation. right? If so, we need to add more code. It seems it is better to have a separate PR for this kind of change.